www/scripts/codecheckerviewer/util.js

```diff
@@ -14,6 +14,18 @@ function (locale, dom, style, json) {
     'July', 'August', 'September', 'October', 'November', 'December'
   ];
 
+  var tooltips = {
+    detectionStatus : "Detection statuses are calculated based on the previous "
```

Comment:
I suggest a small change here:
`Detection statuses are calculated based on the detection status values from the previous store where the reports are stored`